pyqtgraph/GraphicsScene/GraphicsScene.py

```diff
@@ -6,6 +6,9 @@
 from .. import ptime as ptime
 from .mouseEvents import *
 from .. import debug as debug
+from pyqtgraph import getConfigOption
+import time
+get_millis = lambda: int(round(time.time() * 1000))
```

Comment:
1. You can probably accomplish this more simply by using `pg.SignalProxy`
2. Pyqtgraph convention is to use `camelCase` (for consistency with Qt API). I often forget too..